### PR TITLE
postgresql_db: add the rename value to the state option

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Fixes https://github.com/ansible-collections/community.postgresql/issues/100

postgresql_db: add the rename value to the state option

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
postgresql_db

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @asifiqbal @tcraxs @kostiantyn-nemchenko

**Comment by Andersson007** [comment]
cc @andytom

File: plugins/modules/postgresql_db.py [hunk 1]
```diff
@@ -71,8 +71,16 @@
     - Supported compression formats for dump and restore include C(.pgc), C(.bz2), C(.gz) and C(.xz).
     - Supported formats for dump and restore include C(.sql) and C(.tar).
     - "Restore program is selected by target file format: C(.tar) and C(.pgc) are handled by pg_restore, other with pgsql."
+    - "."
```
Comment (tcraxs) [inline plugins/modules/postgresql_db.py:74]: Why is here a `"."`?
I do not understand that, did I missed something?

**Review by tcraxs — APPROVED** [review]
Thanks @Andersson007, I just had this one question, beside that it LGTM

File: plugins/modules/postgresql_db.py (hunk 1 above)
Comment (Andersson007) [reply plugins/modules/postgresql_db.py:74]: it's to add a bit of space between the lines to logically divide them. I tried `-""`  or `-" "` and sanity tests fail. If you know a better way, please share

**Comment by aminvakil** [comment]
needs_rebase

File: plugins/modules/postgresql_db.py (hunk 1 above)
Comment (tcraxs) [reply plugins/modules/postgresql_db.py:74]: Okay, that's fine for me. I do not know a better solution.

**Comment by Andersson007** [comment]
@aminvakil thanks!

**Comment by Andersson007** [comment]
@asifiqbal thanks for the request! @tcraxs @aminvakil thanks for reviewing!

**Pull request merged**